pantheon-bundle/frontend/src/app/__snapshots__/searchBeta.test.tsx.snap

```diff
@@ -296,7 +301,7 @@ exports[`SearchBeta tests should render default Search component 1`] = `
               />
             </DrawerActions>
             <ExpandableSection
-              className=""
+              className="filters-drawer filters-drawer__toggle--by-repo"
```

Comment:
The way these two classes are, it sounds like this one thing is two things. It should either have the `filters-drawer` component class, or have the `filters-drawer__toggle` class, indicating it's part of a component.

I'd go with the first one, since it looks like this is a wrapper. The second class has 'toggle' which sounds like a button to me.

One last note, if there's a `--modifier` suffix at the end of the class, the element should also have the non-suffixed class. e.g.
`filters-drawer__toggle filters-drawer__toggle--by-repo` ✔️ 
`filters-drawer__toggle--by-repo` 🚫